modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -30,4 +29,4 @@ You can install the AWS Load Balancer Operator by using the web console.
 
 .Verification
 
-* Verify that the AWS Load Balancer Operator shows the *Status* as *Succeeded* on the Installed Operators dashboard.
\ No newline at end of file
+* Verify that the AWS Load Balancer Operator shows the *Status* as *Succeeded* on the Installed Operators dashboard.
```

Comment:
I'm not sure what is being changed here, looks like the exact same subtraction and addition.